src/data/data.js

```diff
@@ -1,6 +1,54 @@
 const pageData = {
   home: {
     sections: [
+      {
+        component: 'navigation',
+        image: {
+          src: 'logo.png',
+          alt: 'Logo Ser habil',
+        },
+        logoUrl: 'https://www.google.com',
+        navLinks: [
+          {
+            text: 'inicio',
+            url: 'https://www.google.com',
+          },
+          {
+            text: 'eventos',
+            url: 'https://www.google.com',
+          },
+          {
+            text: 'fundación',
+            url: 'https://www.google.comm',
+          },
+          {
+            text: 'contacto',
+            url: 'https://www.google.comm',
```

Comment:
please remove the, extra  M  in the .com

Reply:
Done